check-shortlog/action.yml

```diff
@@ -0,0 +1,54 @@
+name: Check Shortlog
+description: |
+  Check the output of git shortlog -s matches the list of contributors in the
+  CONTRIBUTING.md file.
+
+inputs:
+  contributing_file:
+    description: Path to contributing file.
+    default: CONTRIBUTING.md
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Get known contributors from contributing file
+      shell: bash
+      run: |
+        echo '::group::Get known contributors from contributing file'
+        sed \
+            -n \
+            -e 's/ - //' \
+            -e 's/^(//' -e 's/)$//' \
+            -e '/start-shortlog/,${p;/end-shortlog/q}' \
+            "${{ inputs.contributing_file }}" \
+            | head -n -1 \
+            | tail -n +2 \
+            | sort \
```

Comment:
I tried running this locally on cylc-flow without the `-n` (quiet) option, but I get basically the whole file in alphabetical order. Would it make sense to wrap the contributors list in a `<div actions:bind="contributors">` like for the changelog date? (Or `<div data-actions-bind="contributors">` if we want to write proper HTML 😛 )

Reply:
Works locally for me.

Reply:
Ah sorry, I didn't check out https://github.com/cylc/cylc-flow/pull/4205